index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+
+<html lang="en"> 
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Apple</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+    <body>
+        <header class="header header-nav">
+            <nav class="header nav-bar">
+                <ul class="nav-bar-ul">
+                    <li class="nav-bar-li"><a href="#" ><img src="images/logo.ico" class="img-logo" alt="logo"></a></li>
+                    <li class="nav-bar-li"><a href="#" >Store</a></li>
+                    <li class="nav-bar-li"><a href="#" >Mac</a></li>
+                    <li class="nav-bar-li"><a href="#" >IPod</a></li>
+                    <li class="nav-bar-li"><a href="#" >IPhone</a></li>
+                    <li class="nav-bar-li"><a href="#" >Ipad</a></li>
+                    <li class="nav-bar-li"><a href="#" >ITunes</a></li>
+                    <li class="nav-bar-li"><a href="#" >Support</a></li>
+                    <li class="nav-bar-li"> <input class="search-icon" type="search"></li>
+                </ul>
+            </nav>
+
+            <h1 class="heading-text"><span class="heading-main">What will you verse be?</span>
```

Comment:
Correct this area of your code with this bullet points

- [ ] There should be three links
- [ ] Each link is an `<a>` HTML tag, with a `<div>` tag inside it.
 - [ ] The `<div>` tag  inside the `<a>` tag should use an image as background,  The text on the original page is actually an image

Reply:
And lastly, please check out the Github workflow, you should be using it on this project and other HTML project, here is a [link](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/)